crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/generate_random_felt.rs

```diff
@@ -0,0 +1,9 @@
+use cairo_vm::Felt252;
+use num_bigint::{BigUint, RandBigInt};
+
+pub fn generate_random_felt() -> Felt252 {
+    let mut rng = rand::thread_rng();
+
+    let random_number: BigUint = rng.gen_biguint(252);
```

Comment:
This can generate values that are not valid felts.

See this

https://book.cairo-lang.org/ch02-02-data-types.html#felt-type

Felts252, despite the name are in the range of `2**251 + some`.

Reply:
I'd just limit it to `rng.gen_biguint(2521);`